competencies/roles/DeveloperLevelMapping.md

```diff
@@ -7,7 +7,7 @@ The three categories of competencies can be used to map to typical job titles sp
 | SW Dev I (junior)         |                                                              |                                |  New Grad, or first one or two years |
 | SW Dev II                 |                                                              |  75% emerging, 25% proficient  | Integrated into the software practice at Properly |
 | SW Dev III (intermediate) | Proficient: "Tech Design, Implementation and Code Ownership" | 50% emerging, 50% proficient   | Strong in knowledge and execution |
-| SW Dev IV (senior I)      | Proficient: "System Design"                                  | 50% proficient; some authority | May be expert or generalist  |
-| SW Dev V (senior II)      | Proficient: "Communication"                                  | 75% proficient; some authority | Expanded impact across teams / org |
-| Principal SW Dev          | Proficient: "Technical Change"                               | 50% Proficient; 50% Authority  | Strong self awareness, Strong on general areas such as business acumen |
+| SW Dev IV (senior I)      | Proficient: "System Design"                                  | 75%  proficient                | May be expert or generalist  |
+| SW Dev V (senior II)      | Proficient: "Communication"                                  | 25% authority                  | Expanded impact across teams / org |
+| Principal SW Dev          | Proficient: "Technical Change"                               | 50% Authority                  | Strong self awareness, Strong on general areas such as business acumen |
```

Comment:
I think the original `50% Proficient; 50% Authority` is worth still keeping here.
This is realistically a `100% Proficient or greater with 50% Authority` which is slightly different than Dev V

Reply:
Agree on wanting to keep this as before `50% Proficient; 50% Authority` for clarity

Reply:
Your highlight here is that, as written one could be emerging in 50% and authority in 50%? 

While I agree that doesn't make sense, I do think that being emerging in an area while being otherwise very strong could be reasonable outcome. I have worked with deep technical experts that needed the business context provided to them to do their work and I think calling that dev 6 could make sense. Thoughts on that scenario?

But agree this is a bit of tangential change to the intended one around dev 5 and dev 6.

Reply:
I agree with @craigatproperly that I could see someone being emerging in one thing but showing really strong authority in most others and still worthy of being a Principal

Reply:
I had assumed that we wanted to be explicit about no emerging in any areas in a principal role, but could be convinced otherwise

Reply:
I guess your starting position @rsiu is that proficiency is something reasonably attainable at this career stage and as defined; and therefore also reasonable expect people to mitigate for this very senior role? (i.e. get a working knowledge of the business even if it isn't your focus)